src/data-table/data-table.styles.tsx

```diff
@@ -35,7 +35,6 @@ export const TableWrapper = styled.div`
     width: 100%;
     border: 1px solid ${borderColor};
     border-radius: 0.5rem;
-    overflow: auto;
```

Comment:
this is needed to constrain the table backgrounds within the border radius, so you'll have to put this back

as for the scrollbar, I found that an alternative fix is to set `overflow: hidden;` on the `LoaderWrapper`. can add a comment in the code if you decide to use this fix